org.civicrm.reqangular/src/common/angular-date/src/services/DateFactory.js

```diff
@@ -1,8 +1,12 @@
 var moment = require('../../../vendor/moment.min.js');
 
-function DateFactory() {
+function DateFactory($q) {
     return {
-        moment: moment,
+        /**
+         * Default Format
+         */
+        dateFormat: 'DD-MM-YYYY',
```

Comment:
I think dateFormat should be `null` at the beginning (see below), while a default/fallback value should be assigned to it only if the asynchronous call fails